manifests/init.pp

```diff
@@ -11,8 +12,12 @@
   $mesh_peerings,    # path to the local peerings description yaml file
 
   $fastd_peers_git,  # fastd peers
+  $fastd_gw_git,     # fastd gateway/backbone peers
   $fastd_secret,     # fastd secret
   $fastd_port,       # fastd port
+  $fastd_gw_port,    # fastd gateway/backbone port
```

Comment:
Can you provide an example for $fastd_gw_port and $fastd_gw_git ?

How do I have to change the gateway.pp?